### PR TITLE
SOC-2131 

### PR DESCRIPTION
**Pull request by qaga**
Updated tests that failed after global release of new auth. Some tests were removed, some updated and those that still needs adjustments have related issue added.

@justnpT

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import com.wikia.webdriver.common.contentpatterns.URLsContent;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.Execute'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.componentobject.dropdowncomponentobject.DropDownComponentObject'.](https://github.com/Wikia/selenium-tests/blob/48e6802039ed8dbc8405cf269747619d35da0e64/src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java#L9) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java [hunk 2]
```diff
@@ -42,21 +42,23 @@ public void staffCanBlockUser() {
...
 
   @Test(groups = {"usersAndRights002"}, dependsOnMethods = {"staffCanBlockUser"})
+  @Execute(asUser = User.BLOCKED_USER)
   public void blockedUserShouldSeeMessageOnArticleEdit() {
     VisualEditModePageObject edit =
         new WikiBasePageObject().goToArticleDefaultContentEditPage(wikiURL,
             PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis());
 
-    edit.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
+    //edit.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java [hunk 3]
```diff
@@ -42,21 +42,23 @@ public void staffCanBlockUser() {
...
 
     edit.verifyBlockedUserMessage();
   }
 
   @Test(groups = {"usersAndRights003"}, dependsOnMethods = {"staffCanBlockUser"})
+  @Execute(asUser = User.BLOCKED_USER)
   public void blockedUserShouldBeAbleToChangeEmail() {
     EditPreferencesPage editPrefPage = new EditPreferencesPage(driver).openEmailSection();
-    editPrefPage.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
+    //editPrefPage.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java [hunk 4]
```diff
@@ -73,7 +73,7 @@ public void userCanLoginViaFacebook() {
     dropDown.openWikiPage(wikiURL);
     dropDown.appendToUrl("noads=1");
     dropDown.openDropDown();
-    dropDown.logInViaFacebook(user.getEmail(), user.getPassword());
+//    dropDown.logInViaFacebook(user.getEmail(), user.getPassword());
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java [hunk 5]
```diff
@@ -190,7 +189,7 @@ public HistoryPagePageObject openFileHistoryPage(String articlePage, String wiki
 
   public SignUpPageObject openSpecialSignUpPage(String wikiURL) {
     getUrl(wikiURL);
-    getGlobalNavigation().signUp();
+    //getGlobalNavigation().signUp();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetSpecialVideosTests.java [hunk 6]
```diff
@@ -38,7 +38,7 @@ public void SpecialVideos_001_Provider() {
 
     FilePagePageObject filePage = new FilePagePageObject(driver).open(video.getWikiFileName());
 
-    filePage.getGlobalNavigation().openAccountNavigation().clickLogOut();
+//    filePage.getGlobalNavigation().openAccountNavigation().clickLogOut();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetSpecialVideosTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**